development/idp-local/config/config.php

```diff
@@ -964,7 +963,7 @@
     /*
      * Which theme directory should be used?
      */
-    'theme.use' => $THEME_USE,
+    'theme.use' => 'material:material',
```

Comment:
Why move from variable to hard-coded?

Reply:
Because we never actually use the default template, which is the only other option for our custom modules. It only led to confusion because some tests used "default" and others used "material". Plus, it allowed me to delete a bunch of files related to the default template.